docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+
+### Why not implement the scheme that the white paper says "miners and clients automatically match orders through order books"？Is it because the LIMITS of TPS?
+
+There will be off-chain orderbooks and miner marketplaces -- some are in development now from some teams. They will work mostly off-chain, because TPS on chain is not enough for the volume of usage we expect on Filecoin. These orderbooks build on the basic deal-flow on chain. These orderbooks will arrive in their own development trajectory -- most likely around or soon after mainnet launch.
+
+### Why does Filecoin mining work best on AMD?
+
+Currently, Filecoin's Proof of Replication (PoRep) prefers to be run on AMD processors. More accurately, it runs much much slower on Intel CPUs (it runs competitively fast on some ARM processors, like the ones in newer Samsung phones, but they lack the RAM to seal the larger sector sizes). The main reason that we see this benefit on AMD processors is due to their implementation of the SHA hardware instructions. Now, why do we use the SHA instruction?
+
+### The Filecoin Client is not usable enough, so how will I give a deal to customers when mainnet launches？
+
+Improving the UX and fixing bugs is our only priority right now. We’re excited to be fixing all the issues the community is reporting, so please keep complaining (file issues on Github, and post in #fil-lotus on Slack).
+
+Clients
+
+### What are the changes to the algorithm and logic layer of lotus and go-filecoin before going online?
+
+There will be no changes to algorithm and logic before we go live -- from here on out, all we’re doing is fixing bugs (so please report them!), improving performance (our sync is much faster on the latest Testnet), and improving the UX with new APIs and documentation.
+
+### How will network retrieval of real data be measured and assessed on an incentivized testnet?
+
+During the competition, once your miner has > 0 storage power, many bots will begin attempting storage and retrieval deals with your miner. The competition dashboard will display your deal success rate in near-real time. Miners below a certain high threshold will be ineligible to receive rewards.
+
+We plan to run the bots for one to two weeks before the competition to give miners a chance to test their configuration, and will announce when that begins.
+
+### What do miners have to do to change a committed capacity (CC) sector into a "real-data" sector?
+
+We are reusing much of the existing code path for this first iteration. Miners will publish storage deals that they will upgrade the CC sector with, announce to the chain that they are doing an upgrade, and prove to the chain that a new sector has been sealed correctly. We expect to evolve and make this cheaper and more attractive over time after mainnet.
+
+### For the incentive phase, you said a miner has to execute the full sector life cycle and terminate the sectors. What does “terminating” the sectors mean?
```

Comment:
For the incentive phase, a miner has to execute the full sector life cycle and terminate the sectors. What does “terminating” the sectors mean?